tests/snapshots/command.help.test.js.md

```diff
@@ -0,0 +1,78 @@
+# Snapshot report for `tests/command.help.test.js`
+
+The actual snapshot is saved in `command.help.test.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## netlify help
+
+> Snapshot 1
+
+    `Netlify command line tool␊
+    ␊
+    VERSION␊
+      netlify-cli/test-version test-os test-node-version␊
+    ␊
+    USAGE␊
+      $ netlify [COMMAND]␊
+    ␊
+    TOPICS␊
+      addons      (Beta) Manage Netlify Add-ons␊
+      completion  (Beta) Generate shell completion script␊
+      dev         Local dev server␊
+      env         (Beta) Control environment variables for the current site␊
+      functions   Manage netlify functions␊
+      lm          Handle Netlify Large Media operations␊
+      open        Open settings for the site linked to the current folder␊
+      plugins     list installed plugins␊
+      sites       Handle various site operations␊
+      status      Print status information␊
+    ␊
+    COMMANDS␊
+      addons      (Beta) Manage Netlify Add-ons␊
+      api         Run any Netlify API method␊
+      build       (Beta) Build on your local machine␊
+      completion  (Beta) Generate shell completion script␊
```

Comment:
We have a custom help class so we can prefix `(Beta)` to the completion plugin and also customize the `help` description to include a not about per command help.